app/views/clients/_form.html.erb

```diff
@@ -16,11 +17,11 @@
   </div>
 
   <div class="form-group">
-    <%= form.label :location %>
+    <%= form.label :Address %>
```

Comment:
address

Reply:
Just change the label. Is there any issue??

Reply:
I guess @taimur-akhtar  here want to highlight incorrect declaration of a symbol. We have symbol in lower case like ` :address`